sources/native/Stride.Native.targets

```
@@ -27,7 +27,7 @@
     <!--<StrideNativeOutputPath>$([MSBuild]::MakeRelative('$(OutputPath)', '$(StridePackageStridePlatformBin)\'))</StrideNativeOutputPath>-->
     <AllowedOutputExtensionsInPackageBuildOutputFolder>.so; .a; $(AllowedOutputExtensionsInPackageBuildOutputFolder)</AllowedOutputExtensionsInPackageBuildOutputFolder>
   
-    <StrideNativeWindowsArm64Enabled Condition="'$(StrideNativeWindowsArm64Enabled)' == ''">true</StrideNativeWindowsArm64Enabled>
+    <StrideNativeWindowsArm64Enabled Condition="'$(StrideNativeWindowsArm64Enabled)' == ''">false</StrideNativeWindowsArm64Enabled>
```

Isn't that change forces to set to True when releasing nuget packages?

---

What do you mean? Do you mean, does this get affected by the team city builds?

---

I .. think so, since by default is true then it's generating arm64 dll files without any additional paramateres. I'm just afraid that's gonna require some additional modification.

---

My proposal would be something like this
```suggestion
<StrideNativeWindowsArm64Enabled Condition="'$(StrideNativeWindowsArm64Enabled)' == '' AND '$(Configuration)' == 'Release'">true</StrideNativeWindowsArm64Enabled>
<StrideNativeWindowsArm64Enabled Condition="'$(StrideNativeWindowsArm64Enabled)' == '' AND '$(Configuration)' != 'Release'">false</StrideNativeWindowsArm64Enabled>
```

---

Would be great if someone else would comment If I'm correct

---

I share your concerns :) But I also understand the intent to make them an opt in feature. Important is that they geht included on TC builds. What about we enable that flag in the build invocation?

---

If it's easy to configure in TC then why, not otherwise I'd go with my change (i.e if value is NaN then set true, then if is not relase, dont geneerate arm files)

---

It could be irritating that the debug build works and once you switch to release it breaks. I'd prefer passing the parameter explicitly to TC build.

---

makes sense

---

Im not very familiar with conditionals in targets files. If we make the change through TC then would my current change be ok currently or would it cause a problem?

---

It should be ok. I think the file we should modify is `build/Stride.build` which contains a property `BuildProperties`, adding `StrideNativeWindowsArm64Enabled=true` should enable it for command line / TC builds.